kuma/wiki/migrations/0001_initial.py

```diff
@@ -285,7 +284,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='document',
             name='team',
-            field=models.ForeignKey(blank=True, to='teamwork.Team', null=True),
+            field=models.ForeignKey(blank=True, to='wiki.DocumentTag', null=True),
```

Comment:
Yeah, that's the plan, squashing migrations is still an important part of the migration system, even after it landed in Django.